tests/test_examples.py

```diff
@@ -27,4 +28,4 @@ def test_example_yamls(fname):
 
     z, pos, batch = create_example_batch()
     model(z, pos, batch)
-    model(z, pos, batch, q=None, s=None)
```

Comment:
Should the case `extra_args={"total_charge": None}` be supported?

Reply:
I don't think so. If that `additional_label` it's None then it's like I don't have it in my data and moreover I cannot train or do inference with None.